ls++

```diff
@@ -362,6 +383,9 @@ sub perm_size_file {
 
 sub perm {
   my ($perm) = @_;
+  if (substr($perm,-1,1) ne "+"){
+    $perm .= " "; # adding an extra space to align with occasional +
```

Comment:
Didn't want to parse the whole permission column to determine if we needed the extra space. For performance reason, I just add a space by default before the column delimiter. It looks good and centered since there is a space at the beginning of the string anyway.